kuma/urls.py

```diff
@@ -91,12 +93,17 @@ def serve_from_media_root(request, path):
                                  include(dashboards_lang_urlpatterns)))
 urlpatterns += [url('users/', include('kuma.users.urls'))]
 urlpatterns += i18n_patterns(
-    url(r'contribute/?$',
-        contribution_views.contribute,
-        name='contribute'),
+    url(r'^payments/$',
+        payment_views.contribute,
+        name='payments'),
+)
+urlpatterns += i18n_patterns(
```

Comment:
We have enough redirects that we have a whole module, ``kuma/redirects/redirects.py``, to handle them. This works, but it should eventually move there.